src/main/java/sh/mob/timer/web/RoomApiController.java

```diff
@@ -85,7 +83,9 @@ public void publishEvent(@PathVariable String roomId, @RequestBody PutTimerReque
           timerRequest.timer,
           timerRequest.user,
           room.name());
-      stats.incrementTimer(timer);
+      if (!Objects.equals(room.name(), SMOKETEST_ROOM_NAME)) {
+        stats.incrementTimer(timer);
```

Comment:
Since its test specific production code, let's minimize it's distraction and move the whole if block into a function `incrementTimerOfSmokeTestRoom` or something, so its only one line.

Reply:
But that’s not what it does! It increments the timers for all room except for the rest room.

Reply:
exactly, sorry 😊